apps/climatemappedafrica/src/components/Footer/index.js

```diff
@@ -1,101 +1,115 @@
 import { QuickLinks, LogoButton, Copyright } from "@commons-ui/core";
 import { Link, StayInTouch } from "@commons-ui/next";
 import { RichText } from "@commons-ui/payload";
-import { Grid } from "@mui/material";
+import { Grid, useMediaQuery } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
 import React from "react";
 
 import useStyles from "./useStyles";
 
 import Section from "@/climatemappedafrica/components/Section";
 
 function Footer(props) {
-  const { title, connect, description, logo: logoProps, links } = props;
+  const {
+    title,
+    connect,
+    description,
+    logo: logoProps,
+    links,
+    variant,
+  } = props;
   const classes = useStyles(props);
+
+  const theme = useTheme();
+
+  const isDesktop = useMediaQuery(theme.breakpoints.up("lg"));
   return (
-    <div className={classes.root}>
-      <Section
-        classes={{
-          root: classes.section,
-        }}
-      >
-        <Grid container direction="row" justifyContent="space-between">
-          <Grid item xs={12} container>
-            {logoProps && (
-              <LogoButton
-                {...logoProps}
-                component="a"
+    !(variant === "explore" && isDesktop) && (
+      <div className={classes.root}>
```

Comment:
Lets use `sx` more...
```suggestion
    <Box sx={{ display: { xs: "none", lg: variant !== "explore" ? "flex" : "none" } }}>
```

Instead of
```jsx
  const theme = useTheme();

  const isDesktop = useMediaQuery(theme.breakpoints.up("lg"));
  !(variant === "explore" && isDesktop) && (
```